common/system/tasks/cli/task_cli_config.cpp

```diff
@@ -188,6 +188,18 @@ void CliTasks::config() {
     }
   }
 
+  // Configure call sign
+  if (options.s) {
+    if (strlen(options.s) != 6) {
+      cli_sendAck(false, "Call sign must be six characters");
```

Comment:
Actually iirc the FCC can assign shorter or longer callsigns. But the basic technician license should be 6 or 7 always. Not positive